src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -102,6 +102,10 @@ class WorkflowManagerActor(dataAccess: DataAccess) extends Actor with CromwellAc
     futureId
   }
 
+  private def loadNamespace(wdlSource: WdlSource): Future[NamespaceWithWorkflow] = {
+    Future(WdlNamespace.load(wdlSource)) map {NamespaceWithWorkflow.from}
```

Comment:
wouldn't it be more straightforward to have a `NamespaceWithWorkflow` method to load this?